CONTRIBUTING.md

```diff
@@ -21,3 +21,12 @@ All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
+
+## What to expect from maintainers
+
+Expect maintainers to respond to new issues or pull requests within a week.
+For outstanding and ongoing issues and particularly for long-running
+pull requests, expect the maintainers to review within a week of a
+contributor asking for a new review. There is no commitment to resolution --
+merging or closing a pull request, or fixing or closing an issue -- because some
+issues will require more discussion than others.
```

Comment:
Please add @rachelmyers as a reviewer to approve the CONTRIBUTING.md change.

Reply:
The contributing guidelines are basically identical to what's present for cel-go. Happy to have @rachelmyers verify.